README.md

```diff
@@ -1 +1,44 @@
-The documentation is on [spilo.readthedocs.org](http://spilo.readthedocs.org/).
+## Spilo: HA PostgreSQL Clusters with Docker
+
+Spilo is a Docker image that provides a resilient, High Available PostgreSQL cluster (HA-cluster) that you can configure and start within minutes. Spilo can run several clusters at one time, with one cluster acting as a master and the others as slaves. Its name derives from სპილო [spiːlɒ], the Georgian word for "elephant." 
+
+Spilo is currently evolving: Its creators are working on a Postgres operator that would make it simpler to deploy scalable Postgres clusters in a Kubernetes environment, and do maintenance tasks. Spilo would serve as an essential building block for this.
+
+Spilo works in combination with [Patroni](https://github.com/zalando/patroni), a template for PostgreSQL HA. Patroni works with [etcd](https://coreos.com/etcd/), [ZooKeeper](https://zookeeper.apache.org/) or [Consul](https://www.consul.io/), and manages the [PostgreSQL](https://www.postgresql.org) databases running in your Spilo cluster. It ensures that there is only one master within your Spilo. (It's worth noting here that Patroni includes a Helm Chart that enables you to deploy a five-node Patroni cluster using a Kubernetes PetSet, in a Google Compute Engine environment.)
+
+###How to Use This Docker Image
+
+First, there are some technical requirements:
+- A Distributed Configuration Store (DCS), to allow distributed nodes to agree. Most Patroni deployments use etcd, which implements the [Raft protocol](https://github.com/coreos/etcd/tree/master/raft). For more information about Raft, visit [this resource](http://thesecretlivesofdata.com/raft/) and check out [this interactive visualization](https://raft.github.io).
+
+>>>**not sure if this is correct?** This image includes 5432 (the Postgres port). The default Postgres user and database are created in the entrypoint with initdb.
+
+#### Administrative Connections//[could this be put as "Connecting to Postgres"?]
+
+You'll need to setup Spilo to create a database and roles for your application(s). For example:
+
+    psql -h myfirstspilo.example.com -p 5432 -U admin -d postgres
+
+#### Application Connections
+
+Once you have created a database and roles for your application, you can connect to Spilo just like you want to connect to any other PostgreSQL cluster:
+
+    psql -h myfirstspilo.example.com -p 5432 -U wow_app -d wow
+    "postgresql://myfirstspilo.example.com:5432/wow?user=wow_app"
+
+#### Connecting to Other Containers
+
+#### Environment Variables
```

Comment:
We have those environment variables that needs to be documented, as without them none will be do anything interesting with WAL-E.

ETCD_HOST - the A record pointing to Etcd hosts. 
ETCD_DISCOVERY_DOMAIN - the SRC record pointing to Etcd hosts.
PGHOME - filesystem path where to put PostgreSQL home directory (/home/postgres by default)
APIPORT - TCP port to Patroni API connections (8008 by default)
BACKUP_SCHEDULE - cron schedule for doing backups via WAL-E (if WAL-E is enabled, '00 01 * * *' by default)
CRONTAB - anything that you want to run periodically as a cron job (empty by default)
PGROOT - a directory where we put the pgdata (by default /home/postgres/pgroot). One may adjust it to point to the mount point of the persistent volume, such as EBS.
WALE_TMPDIR - directory to store WAL-E temporary files.  $PGROOT/../tmp by default, make sure it has a few GBs of free space.
PGDATA - location of PostgreSQL data directory, by default $PGROOT/pgdata.
PGPASSWORD_STANDBY - a password for the standby user, 'standby' by default.
PGPASSWORD_SUPERUSER - a password for the superuser, 'zalando' by default
PGPORT - port PostgreSQL listens to for client connections, 5432 by default
SCOPE - cluster name, multiple Spilos belonging to the same cluster must have identical scope.
SSL_CERTIFICATE_FILE - path to the SSL certificate file inside the container (by default PGHOME/server.crt), Spilo will generate one if not present.
SSL_PRIVATE_KEY_FILE - path to the SSL private key within the container (by default PGHOME/server.key), Spilo will generate one if not present
WALE_BACKUP_THRESHOLD_MEGABYTES - maximum size of the WAL segments accumulated after the base backup to consider WAL-E restore instead of pg_basebackup.
WALE_BACKUP_THRESHOLD_PERCENTAGE - maximum ratio (in percents) of the accumulated WAL files to the base backup to consider WAL-E restore instead of pg_basebackup.
WALE_ENV_DIR - directory where to store WAL-E environment variables
WAL_S3_BUCKET - path to the S3 bucket used for WAL-E base backups (i.e. s3://foobar). Spilo will add /spilo/$scope/wal to that path.
WAL_GCS_BUCKET - ditto for the Google Cloud Storage (WAL-E supports both S3 and GCS).
USE_WALE - whether to consider base backup/restore with WAL-E.
GOOGLE_APPLICATION_CREDENTIALS - credentials for WAL-E when running in Google Cloud.
CALLBACK_SCRIPT - the callback script to run on various cluster actions (on start, on stop, on restart, on role change). The script will receive the cluster name, connection string and the current action. See Patroni documentation for details